analysis/reports/pf_report.Rmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "Pharmacy First"
+output:
+  html_document:
+    toc: true
+    toc_depth: 4
+    theme: journal
+date: "01-01-2020 through 31-06-2024 by month"
+---
+```{css, echo=FALSE}
+pre {
+  max-height: 300px;
+  overflow-y: auto;
+}
+pre[class] {
+  max-height: 200px;
+}
+```
+
+<style>
+  body {
+    text-align: justify;
+     font-size: 11pt}
+</style>
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+
+library(tidyverse)
+library(here)
+library(readr)
+
+
+```
+<br>
+
+## Background <a name="background"></a>
+
+Add background here.
+
+<br>
+
+## Methods <a name="methods"></a>
+
+This study used data from OpenSAFELY-TPP, which covers 40% of the population of England. For a description of the representativeness of this sample, please see our manuscript [here](https://wellcomeopenresearch.org/articles/7-191/v1). Individuals were included if they were alive and registered at a TPP practice each month, across the study period. Patients were excluded if their listed age was not between 0 and 120 years.
```

Comment:
you could have a look at how citations work in R Mardown and see if you can add this paper as a citation? have a look at this if this https://bookdown.org/yihui/rmarkdown-cookbook/bibliography.html if you have time to add, but not essential at this stage